src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_template_builder.py

```diff
@@ -698,7 +701,6 @@ def build_vmss_resource(name, naming_prefix, location, tags, overprovision, upgr
     # Build VMSS
     vmss_properties = {
         'overprovision': overprovision,
-        'singlePlacementGroup': single_placement_group,
```

Comment:
I am fine with different options to address it. But since this PR aims to get into master, then we should not just drop the property, as this clearly makes creating large scale-set no longer work.